src/main/java/webserver/HttpHeaders.java

```diff
@@ -0,0 +1,41 @@
+package webserver;
+
+import java.util.*;
+
+public class HttpHeaders {
+
+    private Map<String, List<String>> loweredKeyValueListMap = new HashMap<>();
+
+    public void addHeader(String k, String v) {
+        loweredKeyValueListMap.compute(k.toLowerCase(), (key, vals) -> {
+            if (vals == null) {
+                vals = new ArrayList<>();
+            }
+
+            vals.add(v);
+            return vals;
+        });
```

Comment:
```suggestion
        loweredKeyValueListMap.computeIfAbsent(k.toLowerCase(), key -> new ArrayList<>()).add(v);
```
요렇게 하면 더 간단해 질것 같아요 😀